src/Esfa.Vacancy.Api.Types/BadRequestResponse.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+
+namespace Esfa.Vacancy.Api.Types
+{
+    /// <summary>
+    /// Error response resulting from a bad request (HTTP STATUS 400)
+    /// </summary>
+    public class BadRequestResponse
```

Comment:
Is this class necessary, we could have returned the collection of BadRequestError

Reply:
Its so that the json response root isn't an array. You'd normally have some kind of containing object first before delving into the array.